src/hotspot/share/jfr/recorder/jfrRecorder.cpp

```diff
@@ -281,6 +281,9 @@ bool JfrRecorder::create_components() {
   if (!create_stacktrace_repository()) {
     return false;
   }
+  if (!create_leak_profiler_stacktrace_repository()) {
```

Comment:
Can you add a comment here explaining the purpose of having a separate leak profiler stacktrace repository?